site2/docs/concepts-messaging.md

```diff
@@ -20,10 +20,11 @@ Value / data payload | The data carried by the message. All Pulsar messages cont
 Key | Messages are optionally tagged with keys, which is useful for things like [topic compaction](concepts-topic-compaction.md).
 Properties | An optional key/value map of user-defined properties.
 Producer name | The name of the producer who produces the message. If you do not specify a producer name, the default name is used. 
-Sequence ID | Each Pulsar message belongs to an ordered sequence on its topic. The sequence ID of the message is its order in that sequence.
+Sequence ID | Each Pulsar message belongs to an ordered sequence on its topic. The sequence ID of a message is assigned by its producer, indicating its order in that sequence.<br />Sequence ID can be used for message deduplication. If `brokerDeduplicationEnabled` is set to `true`, the sequence ID of each message is unique within a partition or a non-partitioned topic. 
```

Comment:
1. The sequence ID also can be assigned by users.
2. The sequence ID of each message is unique within a producer of a topic(nong-partitioned) or a partition

Reply:
Thanks Penghui. By saying "can be assigned by users", does it means that you can manually modify the sequence ID even if it has been auto-assigned by producers?

Reply:
Yes, users can provide the custom sequence ID for a message

Reply:
Penghui, thanks for your clarification. The description of sequence ID has been updated. PTAL.